### DISCUSSION TITLE
Does anybody else have a weird looking lualine after the latest development update of Neovim?

### DISCUSSION THREAD

**Discussion by vicgeentor** [nvim-lualine/lualine.nvim#1309] [Configuration]
<img width="1268" alt="image" src="https://github.com/user-attachments/assets/b889c7ce-6ae5-4250-90c0-6d51d748914a">

The colors are all off.

**Comment by weskoerber** [comment]
I've noticed this as well. I even tracked it down to a specific MR: https://github.com/neovim/neovim/pull/29976.

The MR has the `breaking-change` label, so I doubt it's a bug in Neovim; rather, an update needed within `lualine.nvim`.

**Comment by vicgeentor** [comment]
I found a workaround: see issue [#1312](https://github.com/nvim-lualine/lualine.nvim/issues/1312#issuecomment-2439965065)
